event/register_oracle_event.go

```diff
@@ -21,9 +24,27 @@ func (e RegisterOracleEvent) GetEventAttributeValue() string {
 	return "'RegisterOracle'"
 }
 
-func (e RegisterOracleEvent) EventHandler(event ctypes.ResultEvent) error {
-	// TODO: Verifying Remote Attestation and Executing Vote Txs
+func (e RegisterOracleEvent) EventHandler(event ctypes.ResultEvent, svc *service.Service) error {
+	addressValue := event.Events[types.EventTypeRegistrationVote+"."+types.AttributeKeyOracleAddress][0]
 
-	fmt.Println("RegisterOracle Event Handler")
+	oracleRegistration, err := svc.GrpcClient.GetOracleRegistration(addressValue, hex.EncodeToString(svc.EnclaveInfo.UniqueID))
+	if err != nil {
+		return err
+	}
+
+	err = verifyRemoteReport(oracleRegistration, svc.EnclaveInfo)
+	if err != nil {
+		return err
+	}
```

Comment:
```suggestion
	if err := sgx.VerifyRemoteReport(oracleRegistration.NodePubKeyRemoteReport, oracleRegistration.NodePubKey, *svc.EnclaveInfo); err != nil {
		return err
	}
```

You can use the `sgx.VerifyRemoteReport` directly :)

Reply:
Thanks. Clean code!
Fixed in [f4c315e](https://github.com/medibloc/panacea-doracle/pull/32/commits/f4c315e54e62b38170ac6d964dbddbea9c112c4d).